spec/requests/search_controller_spec.rb

```diff
@@ -330,6 +330,16 @@
       expect(response.status).to eq(400)
     end
 
+    it "doesn't raise an error if the page is a number" do
+      get "/search.json", params: { q: 'kittens', page: '3' }
```

Comment:
Can you add a test for the case of `3 ` or ` 3`, where `.to_i` will return a number that doesn't round trip?

Reply:
👍